drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/exporters/current_waitlists_export.rb

```diff
@@ -33,9 +33,11 @@ def run!
         Array.wrap(groups_by_pool_id[candidate.candidate_pool_id]).each do |unit_group|
           project = unit_group.project
           values = [
+            "#{candidate.id}_#{unit_group.id}",     # ID
```

Comment:
Just to talk it through -- this is "stable" because the candidate id is associated with one candidate pool only. If the Unit Group requirements change, the unit group will be associated to a new candidate pool. even if the same client remains "on the waitlist" for that unit group, they will get a new candidate ID, and so this will be a "new" record.